samples/sensor/icm4268x/src/main.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <stdio.h>
+
+K_SEM_DEFINE(sem_icm42688, 0, 1);	/* starts off "not available" */
+K_SEM_DEFINE(sem_icm42686, 0, 1);	/* starts off "not available" */
+
+static void trigger_handler_icm42688(const struct device *dev, const struct sensor_trigger *trigger)
+{
+	k_sem_give(&sem_icm42688);
+}
+
+static void trigger_handler_icm42686(const struct device *dev, const struct sensor_trigger *trigger)
+{
+	k_sem_give(&sem_icm42686);
+}
+
+int main(void)
+{
+	const struct device *const icm42688 = DEVICE_DT_GET_ONE(invensense_icm42688);
```

Comment:
perhaps use an alias? not required.

Reply:
Seeing - samples/sensor/icm42605/src/main.c.. I feel let it be